scripts/ceph-windows/setup_ceph_vstart

```diff
@@ -43,6 +45,13 @@ time rsync_cmd $WORKSPACE/ceph ${VSTART_DIR}/build-ceph-vstart.sh ${UBUNTU_SSH_U
 time SSH_TIMEOUT=1h ssh_exec ./build-ceph-vstart.sh
 ssh_exec sudo apt-get install -y python3-prettytable
 
+objectstore_args=""
```

Comment:
Is this line needed?  Regardless, I'd use the upper case for `objectstore_args` since all other variables are upper case here.